### PR TITLE
Update Phoenix to 1.7.0-rc.0

### LINKED ISSUE #1118: Refactor to support Phoenix 1.7
https://phoenixframework.org/blog/phoenix-1.7-released

- [ ] Controllers / templates dir structure
- [ ] Refactor views to use new macros / structure
- [ ] Verified Routes

### PR DESCRIPTION
**Pull request by jamilbk**
Fixes #1118

### CODE REVIEW

File: apps/fz_http/lib/fz_http_web/templates/shared/devices_table.html.heex [hunk 1]
```diff
@@ -18,14 +18,14 @@
...
-          <.link navigate={Routes.device_admin_show_path(@socket, :show, device)}>
+          <.link navigate={~p"/devices/#{device}"}>
             <%= device.name %>
           </.link>
         </td>
         <%= if @show_user do %>
           <td>
             <%= live_redirect(device.user.email,
-              to: Routes.user_show_path(@socket, :show, device.user)
+to: ~p"/users/#{device.user}"
```
Comment (akrousset) [inline apps/fz_http/lib/fz_http_web/templates/shared/devices_table.html.heex]: Did the tabbing get squashed here?

**Review by akrousset — DISMISSED** [review]
Left a non-blocking comment about formatting in just one place.

:+1: Looks great, though! The routes look a lot cleaner now. Also glad they're compile time verified, so I scanned over them and trust they're still good :)

File: apps/fz_http/lib/fz_http_web/templates/shared/devices_table.html.heex (hunk 1 above)
Comment (jamilbk) [reply apps/fz_http/lib/fz_http_web/templates/shared/devices_table.html.heex]: Whoops! Looks like the formatter isn’t catching these for some reason

Comment (jamilbk) [reply apps/fz_http/lib/fz_http_web/templates/shared/devices_table.html.heex]: Yep, the formatter wasn't receiving the correct file list. Fixed.

**Pull request merged**